sphinx/source/docs/user_guide/interfaces.rst

```diff
@@ -78,8 +78,8 @@ Bokeh provides a "mid-level" general purpose |bokeh.plotting| interface, which
 is similar in specificity to Matplotlib_ or Matlab_ style plotting interfaces.
 It is centered around having users relate the visual glyphs they would like
 to have displayed to their data, and otherwise taking care of putting together
-plots with sensible default axes, grids, and tools. All of the hard work to
-assemble the appropriate Bokeh Models in the correct way to form a scenegraph
```

Comment:
Not opposed to some change here, but this was intentional -- I am trying to highlight that the act of assembling using the low level glyphs is not trivial.